packages/yew/src/functional/hooks/use_force_update.rs

```diff
@@ -0,0 +1,84 @@
+use super::{Hook, HookContext};
+use crate::functional::ReRender;
+use std::fmt;
+
+/// A handle which can be used to force a re-render of the associated
+/// function component.
+#[derive(Clone)]
+pub struct UseForceUpdate {
```

Comment:
Should we follow `UseXXXHandle` convention and  rename it to `UseForceUpdateHandle`? like `UseReducerHandle` / `UseStateHandle` etc